app/Jobs/FinalizeSlotAttendance.php

```
@@ -0,0 +1,162 @@
+                    $slotService,
+                    $legacyIdService,
+                    $disciplineService,
+                ) {
+                    $attendance = Attendance::query()
+                        ->where('formation_id', $formationId)
+                        ->whereDate('attendance_day', $this->date)
+                        ->lockForUpdate()
+                        ->first();
+
+                    if (! $attendance) {
+                        $attendance = Attendance::create([
```

PHPStan (2.2.6)

[error] Call to an undefined static method App\Models\Attendance::create().


(staticMethod.notFound)